setup/rpi.md

```diff
@@ -96,7 +96,7 @@ First we'll create a workspace for the rover code.
 mkdir -p ~/osr_ws/src && cd ~/osr_ws
 
 # Source your newly created ROS environment
-source /opt/ros/$ROS_DISTRO/setup.bash
+source /opt/ros/melodic/setup.bash
```

Comment:
Don't see why this is necessary. `ROS_DISTRO` is a real variable set when you install ROS, so this is not a placeholder that the user needs to edit.

Reply:
In section "4.1 Setup ROS build environment", at the step "Source your newly created ROS environment", entering the command 
ubuntu@ubuntu:~/osr_ws$ source /opt/ros/$ROS_DISTRO/setup.bash
produces the error message:
"-bash: /opt/ros//setup.bash: No such file or directory"
Replacing the variable "$ROS_DISTRO" with "melodic": 
ubuntu@ubuntu:~/osr_ws$ source /opt/ros/melodic/setup.bash
eliminates the error: